packages/tenant-process/src/services/tenantService.ts

```diff
@@ -150,6 +151,29 @@ export async function retrieveAttribute(
   return attribute;
 }
 
+async function retrieveTenantVerifiedAttribute(
+  tenantId: TenantId,
+  attributeId: AttributeId,
+  readModelService: ReadModelService
+): Promise<{ tenant: WithMetadata<Tenant>; attribute: Attribute }> {
+  const tenant = await retrieveTenant(tenantId, readModelService);
+  const attribute = await retrieveAttribute(attributeId, readModelService);
+  if (attribute.kind !== attributeKind.verified) {
+    throw attributeNotFound(attributeId);
+  }
```

Comment:
IMO there is no need for this check, because if the attribute is not of type verified, the [attributeNotFoundInTenant](https://github.com/pagopa/interop-be-monorepo/pull/2175/files#diff-c52f69aae83f1d150aa67f380d299976f1a67b9cb76738048267062cdb848f3eR171) is already returned (and we spared a query on read model)
```suggestion
```